tutorials/source-modeling/plot_mne_dspm_source_localization.py

```diff
@@ -136,6 +136,10 @@
 brain.add_text(0.1, 0.9, 'dSPM (plus location of maximal activation)', 'title',
                font_size=14)
 
+# The documentation website's movie is generated with:
+# brain.save_movie(..., tmin=0.05, tmax=0.15, interpolation='linear',
+#                  time_dilation=20, framerate=10, time_viewer=True)
```

Comment:
@agramfort the scraper looks for a line that is `# brain.save_movie(` and if it's there actually makes one using the call (equivalently, anyway). So it allows/forces us to keep our narrative doc descriptive while also allowing us to specify parameters.